designer/client/components/Menu/SubMenu.tsx

```diff
@@ -72,7 +72,13 @@ export function SubMenu({ id, updateDownloadedAt }: Props) {
       >
         Download form
       </button>
-      <input ref={fileInput} type="file" hidden onChange={onFileUpload} />
+      <input
+        ref={fileInput}
+        type="file"
+        hidden
+        onChange={onFileUpload}
+        aria-label="country"
```

Comment:
What's the significance of 'country'? This input accepts a form config to upload?

Reply:
aria-label="Import saved form" should now be seeing this